pkg/virt-operator/creation/components/secrets.go

```diff
@@ -20,6 +20,7 @@ import (
 	"kubevirt.io/kubevirt/pkg/certificates/triple/cert"
 )
 
+// #nosec
```

Comment:
Can you please add description of which gosec issue you ignoring here ? Just add it on the `// #nosec ` line